virtual/types/req.go

```diff
@@ -34,4 +34,21 @@ type ActorOptions struct {
 	// that should be created during actor activation.
 	// The value of ExtraReplicas should be a non-negative integer.
 	ExtraReplicas uint64 `json:"extra_replicas"`
+	// RetryPolicy specifies the retry policy for actor invocations.
+	// It defines the behavior when an invocation fails.
+	// Possible values are:
+	// - "retry_never": The invocation will not be retried.
+	// - "retry_if_replica_available": The invocation will be retried on other available replicas.
+	RetryPolicy RetryPolicy `json:"retry_policy"`
 }
+
+// RetryPolicy defines the retry policies for actor invocations.
+type RetryPolicy string
```

Comment:
Performance will never be a concern for this, but I think you should use a struct for the policy. Something like this:

```golang
type RetryPolicy struct {
    Strategy RetryPolicy
    PerAttemptTimeout time.Duration
    MaxNumRetries int
    ...
}
```

This way we can add new options/configuration over time.

PerAttemptTimeout is super important because the user may want to say: "The total context timeout for this is 5s, but if the first replica timesout after 2s try the other one."